### PR TITLE
Azure payg jcayouette

### PR DESCRIPTION
**Pull request by jcayouette**
*Before you submit this PR make sure you have added an entry to **.changelog** file.*

# Description

PAYG Azure documentation.

# Target branches

* Which documentation version does this PR apply to?
* Does this content need to be backported? If this is the case, create an issue for tracking it and add the link to this PR.
* Whenever possible, cross-reference each backport PR here, so that all backports can be easily accessed from the description.

- [x] master
- [x] manager-4.3 (Must be backported!) - https://github.com/uyuni-project/uyuni-docs/pull/2645/

# Links
- This PR tracks issue #<insert spacewalk issue, if any>
- Related development PR #<insert PR link, if any>

### CODE REVIEW

File: modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-webui-configuration.adoc [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+Navigate to the menu:Admin[Task Schedules] to find the **update-payg-default** task. By default, this task is set to run at 10-minute intervals. Its primary function is to monitor the presence of all running PAYG instances by executing the **instance-flavor-check tool**.
+====
+
+
+== Login to the Webui
+
+.Procedure: Login
+// Needs to be updated for azure ask Michael Calmer
+. Login to the SUSE Manager Server WebUI.
+//The username is [FILL THIS IN] the password will be the [FILL THIS IN] on {azure}.
```
Comment (jcayouette) [inline modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-webui-configuration.adoc:25]: @mcalmer Can you verify the username and ID?

Comment (aosthof) [reply modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-webui-configuration.adoc:25]: @jcayouette there's still the old section https://documentation.suse.com/suma/4.3/en/suse-manager/installation-and-upgrade/pubcloud-setup.html in the docs which lists the default user name and password for the WebUI, see "Table 1. Default Administrator Account Details".

File: modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-configure-the-image.adoc [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+. Choose a region for your VM. Ensure it complies with any geo-fencing policies you might have.
+
+. In the **Image** dropdown, select the **manager-server-4-3-ltd** or the **manager-server-4-3-llc** image for your VM. If the image is not listed, select **Browse all public and private images** to find the required image.
+ 
+ . Suggested sizes are selected by default for this VM. Configure optional features such as virtual CPUs, memory as required.
+
+. Set up an administrator account for the VM. For {sles}, this can be a username and SSH public key.
+
+. Ensure the VM is connected to the appropriate virtual network (VNet) and subnet, especially if it needs to communicate with other specific services like {productname}.
+For network requirements, see: xref:public-cloud-guide/payg/azure/payg-azure-server-setup.adoc[Azure Server Setup: Network Configuration]
```
Comment (keichwa) [inline modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-configure-the-image.adoc:27]:
```suggestion
For network requirements, see xref:specialized-guides:public-cloud-guide/payg/azure/payg-azure-server-setup.adoc[Azure Server Setup: Network Configuration].
```

File: modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-public-cloud-images.adoc [hunk 3]
```diff
@@ -0,0 +1,25 @@
+= Obtaining the {productname} Server {payg} public cloud image on {azure}
+
+Follow these step-by-step instructions to locate the SUSE Manager {productnumber} {payg} image on {azure}.
+You can also review the latest available images for the public cloud using Pint(Public Cloud Information Tracker).
```
Comment (keichwa) [inline modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-public-cloud-images.adoc:4]:
```suggestion
You can also review the latest available images for the public cloud using Pint (Public Cloud Information Tracker). 
```

File: modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-webui-configuration.adoc [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+[IMPORTANT]
+====
+.Ubuntu and Debian
+Ubuntu and Debian are not currently supported by our RMT servers.
+
+.Red Hat Linux Enterprise Clients
+{productname} offers support for {rhel} clients; however it is important to note that content still needs to be added manually and is not available through the cloud-based RMT service.
+
+For more information see: xref:client-configuration:registration-overview-redhat.adoc[Red Hat Client Registration].
```
Comment (keichwa) [inline modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-webui-configuration.adoc:57]:
```suggestion
For more information, see xref:client-configuration:registration-overview-redhat.adoc[Red Hat Client Registration].
```

**Review by keichwa — APPROVED** [review]
Would nice if you could commit my suggestions.  Thanks!

File: modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-configure-the-image.adoc [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+. Configure additional settings such as storage, monitoring, or any extensions you may need. The following partitions are created by default when initializing the {azure} image:
+
+* **200 GB** for the root partition
```
Comment (mcalmer) [inline modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-configure-the-image.adoc:31]: I think this is just `100 GB`

File: modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-requirements.adoc [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+By default, certain disks are automatically generated when establishing a {productname} {payg} instance on {azure}. To complete the setup of these disks, use the **suma-storage** tool.
+
+.Azure System Requirements
+[cols="2,1"]
+|===
+| Requirement | Details
+
+| root storage
+| 200 GB
```
Comment (mcalmer) [inline modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-requirements.adoc:12]: `100 GB`

File: modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-server-setup.adoc [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+
+== Configure Storage
+. Configure storage with the **suma-storage** tool. 
+** For help see: `suma-storage --help`
+This tool simplifies creating the storage disk device and database device.
+
+. Use `lsblk` to see the names of your disk devices.
+You will see a list of partitions that were created when we configured the {azure} image.
+
+** Root partition 200 GB
```
Comment (mcalmer) [inline modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-server-setup.adoc:47]: 100 GB

**Review by mcalmer — CHANGES_REQUESTED** [review]

File: modules/specialized-guides/nav-specialized-guides-guide.adoc [hunk 8]
```diff
@@ -16,12 +16,19 @@ endif::[]
...
-***** xref:public-cloud-guide/payg/payg-configure-the-image.adoc[2. Configuration and Startup]
-***** xref:public-cloud-guide/payg/payg-server-setup.adoc[3. Server Setup]
-***** xref:public-cloud-guide/payg/payg-webui-configuration.adoc[4. WebUI Configuration]
-***** xref:public-cloud-guide/payg/payg-connections-and-bootstrapping.adoc[5. Connecting Clients and Client Registration]
+***** xref:public-cloud-guide/payg/aws/payg-aws-requirements.adoc[AWS requirements]
+***** xref:public-cloud-guide/payg/aws/payg-aws-public-cloud-images.adoc[1. Public Cloud Images]
+***** xref:public-cloud-guide/payg/aws/payg-aws-configure-the-image.adoc[2. Configuration and Startup]
+***** xref:public-cloud-guide/payg/aws/payg-aws-server-setup.adoc[3. Server Setup]
+***** xref:public-cloud-guide/payg/aws/payg-aws-webui-configuration.adoc[4. WebUI Configuration]
+***** xref:public-cloud-guide/payg/aws/payg-aws-connections-and-bootstrapping.adoc[5. Connecting Clients and Client Registration]
```
[the remark this answers is not in the document]
Comment (jcayouette) [reply modules/specialized-guides/nav-specialized-guides-guide.adoc:24]: No, the references given to the cloud team have remained the same. The files I referenced are in the root payg directory.

[the remark this answers is not in the document]
Comment (mcalmer) [reply modules/specialized-guides/nav-specialized-guides-guide.adoc:24]: :+1:  Thanks

File: modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-server-setup.adoc [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+----
+sudo -i
+zypper refresh
+zypper update
+----
+
+== Network Configuration
+
+Before initiating `yast2 susemanager_setup`, ensure the network is correctly configured for {azure}.
+
```
Comment (mcalmer) [inline modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-server-setup.adoc:19]: I would add here , that in Azure for the SUMA PAYG setup it is required, that a new Virtual Network and a new subnet is configured. (Azure requirement, not our).
So it is impossible to setup the instance in an existing network.

When you setup new clients which should be managed, take care that you put them into the private subnet which is configured with SUMA PAYG. (This is the section below).

If the customer want to manage systems in an already existing network, they can configure "peering".

File: modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-requirements.adoc (hunk 6 above)
Comment (0rnela) [inline modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-requirements.adoc:12]:
```suggestion
| 100 GB
```

File: modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-configure-the-image.adoc (hunk 5 above)
Comment (aosthof) [reply modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-configure-the-image.adoc:31]: Should also be changed in the AWS specific section to `100GB` for the root partition.

File: modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-requirements.adoc (hunk 6 above)
Comment (aosthof) [reply modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-requirements.adoc:12]: Should also be changed in the AWS specific section to `100GB` for the root partition.

File: modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-server-setup.adoc (hunk 7 above)
Comment (aosthof) [reply modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-server-setup.adoc:47]: Should also be changed in the AWS specific section to `100GB` for the root partition.

**Comment by keichwa** [comment]
I now try to add the other suggestions by mc and aosthof.

File: modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-configure-the-image.adoc (hunk 5 above)
Comment (keichwa) [reply modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-configure-the-image.adoc]: ok done here, too

File: modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-server-setup.adoc (hunk 9 above)
Comment (keichwa) [reply modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-server-setup.adoc:21]: Thanks, I added that note.  I hope I got it right.

**Review by 0rnela — APPROVED** [review]

**Review by mcalmer — APPROVED** [review]
Looks good for the first shot

**Review by aosthof — APPROVED** [review]
LGTM

**Comment by keichwa** [comment]
Thanks!  I'll merge it.
Then we probably should backport it to 4.3, 4.3.9, and 4.3.10. Update external-tree with 4.3.9, and ask toms (doc team) to publish it on d.suse.com.

**Comment by jcayouette** [comment]
Thank you everyone for working over the remaining pieces. Best!

**Comment by keichwa** [comment]
> Thank you everyone for working over the remaining pieces. Best!

No worries, preparation and all the feedback was very helpful!

**Pull request merged**